shared/reducers/signup.js

```diff
@@ -33,6 +33,7 @@ const initialState: SignupState = {
   phase: 'inviteCode'
 }
 
+/* eslint-disable no-fallthrough */
```

Comment:
annoyingly eslint gets confused here and if we do add breaks then flow complains. i *think* eslint 2.0 fixes this but its not out of rc yet